stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -729,6 +805,17 @@ static String getRetrieveTokenApiUrl(@NonNull String tokenId) {
         return String.format(Locale.ROOT, "%s/%s", getApiUrl(), tokenId);
     }
 
+
+    @VisibleForTesting
+    static String getRetrieveIssuingCardPinUrl(@NonNull String cardId) {
```

Comment:
is it expected that `getRetrieveIssuingCardPinUrl` and `getUpdateIssuingCardPinUrl` return the same string?